ci/licenses_golden/licenses_third_party

```diff
@@ -2176,6 +2180,67 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
 OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
+====================================================================================================
+LIBRARY: boringssl
+ORIGIN: ../../../third_party/boringssl/src/crypto/err/err.c
+TYPE: LicenseType.bsd
+FILE: ../../../third_party/boringssl/src/crypto/err/err.c
+FILE: ../../../third_party/boringssl/src/crypto/fipsmodule/bn/internal.h
+FILE: ../../../third_party/boringssl/src/crypto/fipsmodule/bn/montgomery.c
+FILE: ../../../third_party/boringssl/src/crypto/fipsmodule/rsa/blinding.c
+FILE: ../../../third_party/boringssl/src/include/openssl/bn.h
+FILE: ../../../third_party/boringssl/src/include/openssl/err.h
+FILE: ../../../third_party/boringssl/src/include/openssl/tls1.h
+FILE: ../../../third_party/boringssl/src/ssl/d1_srtp.cc
+FILE: ../../../third_party/boringssl/src/ssl/ssl_session.cc
+----------------------------------------------------------------------------------------------------
+Copyright (c) 1998-2006 The OpenSSL Project.  All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in
+   the documentation and/or other materials provided with the
+   distribution.
+
+3. All advertising materials mentioning features or use of this
+   software must display the following acknowledgment:
+   "This product includes software developed by the OpenSSL Project
+   for use in the OpenSSL Toolkit. (http://www.openssl.org/)"
```

Comment:
is this a new clause in our current LICENSE file, or do we have this clause already?

Reply:
never mind